snapcraft/internal/errors.py

```diff
@@ -531,3 +531,17 @@ class InvalidExtractorValueError(MetadataExtractionError):
 
     def __init__(self, path: str, extractor_name: str) -> None:
         super().__init__(path=path, extractor_name=extractor_name)
+
+
+class SnapcraftPluginCommandError(SnapcraftError):
+    """Command executed by a plugin fails."""
+
+    fmt = (
+        'The command {command!r} run for the {part_name!r} part has failed.\n'
+        'Verify that the part is using the correct parameters and try again.'
```

Comment:
I think the exit code would be helpful as well. Perhaps something like:

```
'Failed to run {command!r} for {part_name!r}: '
'Exited with code {exit_code}.\n'
'Verify that the part is using the correct parameters and try again.'
```

Reply:
done